cinn/hlir/op/custom_call.cc

```diff
@@ -131,6 +131,7 @@ std::vector<ir::Expr> CustomCallArgsForCublas(const framework::NodeAttr &attrs,
 
   int x_num_col_dims = attr_store.count("x_num_col_dims") ? absl::get<int>(attr_store.at("x_num_col_dims")) : 0;
   int y_num_col_dims = attr_store.count("y_num_col_dims") ? absl::get<int>(attr_store.at("y_num_col_dims")) : 0;
+  int is_infer       = attr_store.count("is_infer") ? absl::get<bool>(attr_store.at("is_infer")) : false;
```

Comment:
Should we declare `bool is_infer`? 

If we cannot do that due to `attr_store`, should we change `absl::get<bool>` to `absl::get<int>` ?

Reply:
Sorry, here should be `bool`.